backend/typescript/package.json

```diff
@@ -7,7 +7,8 @@
     "test": "jest --runInBand --forceExit --detectOpenHandles",
     "dev": "nodemon -L",
     "lint": "eslint . --ext .ts,.js",
-    "fix": "eslint . --ext .ts,.js --fix"
+    "fix": "eslint . --ext .ts,.js --fix",
+    "seed": "ts-node prisma/dummy_users.ts"
```

Comment:
let's not commit this `seed` command since we're not committing `dummy_users.ts`